src/metadata.rs

```diff
@@ -1,13 +1,13 @@
-use num_derive::FromPrimitive;
+use num_derive::{FromPrimitive, ToPrimitive};
 use serde::{Deserialize, Serialize};
-use std::convert::TryFrom;
+use std::{convert::TryFrom, fmt};
 
 /// ProtocolVersion describes the version of the communication protocol
 /// used to exchange information between the policy and the policy evaluator.
 ///
 /// Policies built with this SDK provide the right value via the `protocol_version_guest`
 /// function.
-#[derive(Deserialize, Serialize, Debug, Clone, FromPrimitive, PartialEq)]
+#[derive(Deserialize, Serialize, Debug, Clone, FromPrimitive, ToPrimitive, PartialEq)]
```

Comment:
I guess the `Display` derived version is not what we want and that's why we have to implement Display ourselves?

Reply:
There's no derived Display trait available, we have to implement it ourselves.